src/pytypes/onefuzztypes/models.py

```diff
@@ -835,9 +840,8 @@ class InstanceConfig(BaseModel):
     allow_pool_management: bool = Field(default=True)
 
     allowed_aad_tenants: List[UUID]
-
+    network_config: NetworkConfig = Field(default=NetworkConfig())
```

Comment:
It probably shouldn't matter in practice, but I think we really want `default_factory=NetworkConfig`, or similar.